matlab-src/whole_body_model_functions/wbm_getWorldFrameFromFixedLink.m

```diff
@@ -0,0 +1,33 @@
+function [ world_R_base,world_p_base ] = wbm_getWorldFrameFromFixedLink( varargin )
```

Comment:
The ordering in most of the wbi and mex-wholebodymodel is linear-angular ( see https://github.com/robotology/mex-wholebodymodel/blob/6603ab444e828f05bcb71c0317e4421e97736ac8/matlab-src/worker_functions/frame2posrot.m ) but here is angular-linear instead ... I see way, apparently `wbm_setWorldFrame` is also angular-linear . Ok for now, but it is quite error-prone.  
@gabrielenava what you think about this?